File: 第３章/3_6_copilot_lab_brushes_sample.js
function doSomething(x, y) {
    let temp = []
    for (let i = x; i < y; i++) {
        if (i % 3 == 0) {
            temp.push(i)
        } else {
            continue
        }
    }
    for (let j = 0; j < temp.length; j++) {
        temp[j] = temp[j] * 3
    }
    let result = 0
    for (let k = 0; k < temp.length; k++) {
        result = result + temp[k]
    }
    return result
}
console.log(doSomething(1, 15));

File: 第３章/3_6_copilot_lab_brushes_sample.py
def do_something(start: int, end: int) -> None:
    multiples_of_three: List[int] = []
    for num in range(start, end):
        if num % 3 == 0:
            multiples_of_three.append(num)
    multiples_of_three = [num * 3 for num in multiples_of_three]
    result: int = sum(multiples_of_three)
    print(result)


do_something(1, 15)
